### PR TITLE
Store feedback survey responses

### PR DESCRIPTION
**Pull request by tej99**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
   - Add codacy badge
   - Add support for codecov to see unit test coverage
   - Update and pin version of sdx-common to 0.7.0
+  - Feedback survey responses routed to another table
```
Comment (necrophonic) [inline CHANGELOG.md:13]: Should be in imperative tense: e.g. "Add support for feedback survey responses"

File: server.py [hunk 2]
```diff
@@ -141,13 +170,13 @@ def merge(response):
...
-    except SQLAlchemyError as e:
-        logger.info("Unable to save response", error=e)
-        raise SQLAlchemyError
     except IntegrityError as e:
         logger.info("Integrity error in database. Rolling back commit",
                     error=e)
-        raise IntegrityError
+        raise
+    except SQLAlchemyError as e:
+        logger.info("Unable to save response", error=e)
```
Comment (necrophonic) [inline server.py:178]: Every log line should either use a logger with `tx_id` prebound or should specifically log `tx_id`

File: server.py [hunk 3]
```diff
@@ -175,6 +204,33 @@ def save_response(bound_logger, survey_response):
...
+    feedback_response = FeedbackResponse(invalid=invalid,
+                                         data=survey_feedback_response,
+                                         survey=survey,
+                                         period=period)
+
+    try:
+        db.session.add(feedback_response)
+        db.session.commit()
+    except IntegrityError as e:
+        logger.info("Integrity error in database. Rolling back commit",
```
Comment (necrophonic) [inline server.py:222]: Need to be logging `tx_id`

File: server.py (hunk 2 above)
Comment (JamesGardiner) [reply server.py:178]: Aren't we keeping these anonymous - i.e. no tx_id?

**Comment by necrophonic** [comment]
Closing for now - will be reopened following further work and onto `master` branch

**Pull request closed without merging**